loom/graphics/gfxVectorRenderer.cpp

```diff
@@ -451,35 +454,41 @@ void VectorSVG::resetInfo() {
         path = NULL;
     }
 }
+
 void VectorSVG::resetImage() {
     if (image != NULL) {
         nsvgDelete(image);
         image = NULL;
     }
 }
+
 void VectorSVG::loadFile(utString path, utString units, float dpi) {
 	reset();
-	this->units = utString(path);
+	this->units = units;
 	this->dpi = dpi;
 	this->path = lmNew(NULL) utString(path);
 	reload();
-	loom_asset_subscribe(path.c_str(), onReload, this, false);
+	loom_asset_subscribe(path.c_str(), onReload, this, true);

```

Comment:
Doesn't `reload();` before it take care of that already? I think it should be either that the `reload();` is there or that `loom_asset_subscribe` is `true`, but not both?